index.html

```diff
@@ -243,18 +286,19 @@ <h2>
         </section>
         <section class="text">
             <p>
-                I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
-                don’t hesitate to contact me.
+                I can help you build a product , feature or website Look through some of my work and experience! 
+                If you like what you see and have a project you need coded, don’t hesitate to contact me.
             </p>
         </section>
-        <form class="connect" action="https://formspree.io/f/mwkjdnnq" method="POST" autocomplete="off">
+        <form class="connect" id="connect" action="https://formspree.io/f/mwkjdnnq" method="POST" autocomplete="off">
             <label for="name">Full name</label>
-            <input id="name" name="username" placeholder="Full name" required type="text" maxlength="30"/>
+            <input id="name" name="username" placeholder="Full name" type="text" maxlength="30" required/>
             <label for="mail">Email</label>
-            <input id="mail" name="mail" placeholder="Email adress" required type="email"/>
+            <input id="mail" name="mail" placeholder="Email adress" type="email" required/>
```

Comment:
- [ ] You have done really great job in making the changes, However, Please ensure that the email validation test against Upper cases as well,  as required by the project requirement. It should throw an error if the email contains any uppercase letters. :slightly_smiling_face: 

Below is a screenshot of yours not capturing the error message of uppercases: :arrow_down: 

![Screenshot from 2023-06-22 18-19-17](https://github.com/Arch-Noize/portfolio-microverse/assets/43528594/c81c1921-5784-41c9-b7f2-74e3386d85bc)

Reply:
Hi! If I'm not mistaken, that's because I ask for the user to fill everything out before they can send their info. If the comment section is filled, it should bring out the error.

Should I make it an immediate check rather than a check done by submission?

Reply:
![image](https://github.com/Arch-Noize/portfolio-microverse/assets/91759364/eacac748-fe13-4307-b168-952bea289dcf)

This is what I mean!

Reply:
Hello @Arch-Noize, I think you should commit your changes, because it still sends successful message from here.

see your screenshot below: :arrow_down: 

![Screenshot from 2023-06-22 18-57-08](https://github.com/Arch-Noize/portfolio-microverse/assets/43528594/fe5f5412-fe4e-4165-ad55-89239603ffe3)